File: unsorted/defangIPaddr.test.js
import defangIPaddr from "../src/defangIPaddr.js";
test.each([
  ["1.1.1.1", "1[.]1[.]1[.]1"],

  ["255.100.50.0", "255[.]100[.]50[.]0"],
])("defangs ip address", (test_input, expected) => {
  let func = defangIPaddr(test_input);
  expect(func).toBe(expected);
});

File: unsorted/defangIPaddr_test.py
import pytest

from python_refactored_waddle.defangIPaddr import defangIPaddr


@pytest.mark.parametrize(
    "test_input,expected",
    [("1.1.1.1", "1[.]1[.]1[.]1"), ("255.100.50.0", "255[.]100[.]50[.]0")],
)
def test_defangIPaddr(test_input, expected):
    assert defangIPaddr(test_input) == expected
